openjdk.test.debugging/build.xml

```diff
@@ -113,6 +145,9 @@ limitations under the License.
 			   encoding="${src-encoding}"
 			   includeantruntime="false"
 			   failonerror="true">
+			<compilerarg value="-source"/>
+			<compilerarg value="${java.specification.version}"/>
+			<compilerarg value="--enable-preview"/>
```

Comment:
Is this argument ignored in the jdk11 case (as its in the build-java-11-and-up target)?

Reply:
Are you spotting anything in the logic that may prevent  `build-java-11-and-up` from running on JDK 11?. I tested it with JDK 8 and JDK 14 only, did not test it with JDK 11. `build-java-11-and-up` is meant only be executed when we are not running JDK 8 (i.e., running JDK 11 and up - i.e., only the JDKs that support `--enable-preview` option).

Reply:
I just understood --enable-preview to essentially exist to enable preview features in tip (which will be ever-changing, jdk14... then jdk15, etc).  It looked like its being added as an arg to everything jdk11 and up, so was wondering what effect it had in jdk11, which is not tip... and there would be no features to preview.  As long as its benign, we are good to merge.